src/tools/Makefile-client.am

```diff
@@ -72,6 +72,7 @@ noinst_HEADERS += \
 rbd_LDADD = \
 	libjournal.la libcls_journal_client.la \
 	libcls_rbd_client.la libcls_lock_client.la \
+	librbd_internal.la \
```

Comment:
We probably don't want to pull in librbd.so and librbd_internal.a.  While I don't like the idea of duplicate code, I think you just need to have an rbd CLI version and a librbd version of "parse_rbd_default_features".